### DISCUSSION TITLE
Filter options in Price Listing page.

### DISCUSSION THREAD

**Discussion by avinashkanaujiya** [ananthakumaran/paisa#100] [Ideas]
Few options for filtering could be
- Portfolio - show only those that are in portfolio currently
- Mutual Funds
- Stocks

**Comment by ananthakumaran** [comment]
> Portfolio - show only those that are in portfolio currently

You mean the commodity balance > 0

**Reply by avinashkanaujiya** [reply]
If, by that, you mean I own it currently, then yes.
